.gitignore

```diff
@@ -44,3 +44,6 @@ testem.log
 # System Files
 .DS_Store
 Thumbs.db
+
+#angular
```

Comment:
Can be cleaned up

Reply:
Hello

Sall I leave the rule though ? because this folder exlcusion is pretty usefull and it seems to be by default in a fresh install, so I removed line 47/48